scripting/get5_apistats.sp

```diff
@@ -196,17 +208,31 @@ static void CheckForLogo(const char[] logo) {
   }
 }
 
-static int LogoCallback(Handle request, bool failure, bool successful, EHTTPStatusCode status,
-                        int data) {
+static int LogoCallback(Handle request, bool failure, bool successful, EHTTPStatusCode statusCode,
+                        DataPack data) {
   if (failure || !successful) {
-    LogError("Logo request failed, status code = %d", status);
+    LogError("Network connection failed for the logo request");
```

Comment:
Missing full stop. I would also recommend you move the `DataPack` logic up to the beginning of this method, so you can error out which request failed, i.e:

```c
LogError("Network connection failed for the request for logo at '%s'.", logo);
```

You are leaking memory now if the request fails, as you are not deleting the datapack at all then.